test/sequential/test-http2-session-timeout.js

```diff
@@ -34,12 +34,14 @@ server.listen(0, common.mustCall(() => {
     request.end();
 
     request.on('end', () => {
-      if (attempts) {
-        setTimeout(() => makeReq(attempts - 1), callTimeout);
+      const diff = process.hrtime(startTime);
```

Comment:
I'm -1 on this change. `process.hrtime` isn't cheap so it's more — not less — likely to create situations where this test fails on slow platforms.